concent_api/utils/helpers.py

```diff
@@ -2,14 +2,30 @@
 import base64
 import binascii
 import datetime
+import os
 import time
 
+from django.conf                    import settings
+from django.core.checks             import Error
+from django.core.checks             import register
 from django.utils                   import timezone
 
 from golem_messages                 import message
 from golem_messages.datastructures  import FrozenDict
 
 
+@register
+def storage_cluster_certificate_path_check(app_configs = None, **kwargs):
+    errors = []
+    certificate_path = settings.STORAGE_CLUSTER_SSL_CERTIFICATE_PATH
+    if certificate_path != '':
+        if not os.path.exists(certificate_path):
+            errors.append(Error("File not found"))
```

Comment:
This message does not give user enough detail to realize which file is being referred too.

Here's how the output of `manage.py check` looks like if you use a non-existent path right now:
```
SystemCheckError: System check identified some issues:

ERRORS:
?: File not found

System check identified 1 issue (0 silenced).
```

Reply:
done